test/openTerm.js

```diff
@@ -0,0 +1,35 @@
+const { existsSync, unlink } = require("fs");
+const { assert } = require("chai");
+const { after } = require("mocha");
+const log = require("electron-log");
+const openTermLinux = require("../app/electron/platform/linux/openTerm");
+const openTermWin = require("../app/electron/platform/windows/openTerm");
+
+after(() => {
+  unlink("./test.txt", (err) => {
+    if (err) log.error(err);
+    else log.info("Test completed and file deleted");
```

Comment:
Always use {} for if-else, even when not required, usually improves readability. Also, don't log this here, as this will be logged in electron log file, which should not contain test logs

Reply:
im not allowed by eslint to use the default console because we have added this [rule](https://eslint.org/docs/rules/no-console). I researched a bit but found some [code](https://stackoverflow.com/questions/31759367/using-console-log-in-electron-app#answer-48923571) doing the same.